functions/configure_events.ts

```diff
@@ -5,11 +5,13 @@ import {
   findReactionTriggers,
   getReactionTriggerChannelIds,
   getReactionTriggerSurveyorIds,
+  ReactionTriggerResponseObject,
   updateReactionTriggers,
 } from "./utils/trigger_operations.ts";
+import { isTriggerOperationError } from "./utils/errors.ts";
 
 /**
- * Custom funcitons are the building blocks of workflows:
+ * Custom functions are the building blocks of workflows:
  * accepting input, performing calculations, handling view events,
  * and providing output.
  * https://api.slack.com/future/functions/custom
```

Comment:
```suggestion
 * https://api.slack.com/future/automation/custom
```

Can you make sure all of these `/future/` references are switched to `/automation/`?

Reply:
Done 🟢